examples/nodejs/cache/.eslintrc.json

```diff
@@ -51,7 +51,8 @@
     // the intent of the developer. Functions are async when they want to await.
     "require-await": "error",
     "import/no-duplicates": "error",
-    "@typescript-eslint/switch-exhaustiveness-check": "error"
+    "@typescript-eslint/switch-exhaustiveness-check": "error",
+    "@typescript-eslint/no-non-null-assertion": "off"
```

Comment:
Will we include these details in the example README?

Reply:
we definitely should, i will add some notes in the tickets